Packs/QueryAI/Integrations/QueryAI/QueryAI.py

```diff
@@ -181,12 +182,11 @@ def main() -> None:
     :rtype:
     """
 
-    email = demisto.params().get('email')
-    license_key = demisto.params().get('license_key')
+    api_token = demisto.params().get('api_token')
     base_url = urljoin(demisto.params()['url'], '/api/v1')
     alias = demisto.params().get('alias')
     connection_params = demisto.params().get('connection_params', '{}')
-    timeout = int(demisto.params().get('timeout', DEFAULT_TIMEOUT))
+    timeout = int(demisto.params().get('timeout', DEFAULT_TIMEOUT)) if demisto.params().get('timeout') else DEFAULT_TIMEOUT
```

Comment:
you don’t need to add this

Reply:
I know but for some strange reason this wasn't working now and 'timeout' was still None, so I explicitly added a fail-safe.